v1.0/Specification.md

```diff
@@ -183,7 +183,7 @@ The following table gives guidelines for how to comply with the requirements in
 | ODS_4 | Any external data references such as data connections, RTD functions, cell references or objects can be removed, however the calculated cell value MUST be preserved.<br><br>Removal of data connections will not result in any risk in loss of calculated cell values.<br><br>Removal of RealTimeData functions and cell references by removing the underlying formula must happen by simultaneously preserving the calculated cell value.<br><br>External object references may be included as an independent object and child/parent relationships created. Thirdly, external object references can be removed with an appraised acceptance of the data loss.<br><br> Cell hyperlinks are exempt and covered in ODS_11.<br><br> Removed external data can be saved as metadata in a sidecar file or in an associated table, to document its existence before removal. |
 | ODS_5 | Embedded objects can be converted to a file format, your organisation accepts. Secondly, embedded objects can be removed with an appraised acceptance of the data loss. |
 | ODS_6 | A spreadsheet without any cell values or objects is most likely empty of content. To easily check this, read the “meta.xml” and check if “meta:cell-count” or “meta:object-count” is greater than 0.<br><br>If no cell values or objects exist, the file must be appraised to determine if other kind of content exists. If so the file may be preserved.<br><br>Information may be stored in user defined properties or color formatting of cells may have been used to semantically convey a message. These are potential occurences, which after appraisal could lead to preserving the spreadsheet. |
-| ODS_7 | Macros contain programming code that can execute and manipulate data in the spreadsheet and the file system. These may present security risks, and it is therefore prudent to remove any macros. Removal may result in loss of information, but typically macros are used to change data, and this goes against the intentions of preserving the data fixed in time, and removal may therefore often happen with no implications. |
+| ODS_7 | Macros contain programming code that can execute and manipulate data in the spreadsheet and the file system. These may present security risks, and it is therefore prudent to remove or disable any macros. Macros add interaction to a spreadsheet and can change data.<br><br>Removing macros will prevent any data from changing, but the macro itself will be lost if not documented other places.<br><br>Another option is to set up your spreadsheet rendering application to disable any macros. This will prevent any data from changing, it will preserve the interactivity of the macro and the macro may be reused by a user to support original reuse scenarios, but the same interactivity may also lead to change of data in unintended and non-transparent ways, if the user is not careful. |
```

Comment:
I feel that the requirement is becoming way too long here in an attempt to present the rationale. The hints as to what an archivist might do about macros don't belong in the requirement.

Reply:
It's not the requirement/policy, it is the guideline section.

Reply:
I'm going to approve it for now, but the text is too long and rambling. I will catch it in a full document review.